stor/dx.py

```diff
@@ -340,7 +350,7 @@ def _rename(self, new_name):
             NotFoundError: When path cannot be resolved to a file.
         """
         if not self.resource:
-            raise ValueError('Projects cannot be renamed')
+            raise ValueError('Paths without resource cannot be renamed')
```

Comment:
I think the original message is actually fine (and clearer for 90% of all cases).  or do an if statement:

```
if not self.resource:
   raise ValueError('Projects cannot be renamed' if self.project else 'Cannot rename dx root')
```

Reply:
Changed